src/components/button/Button.js

```diff
@@ -45,6 +46,7 @@ class Button extends React.Component<Props> {
             setRef,
             type = 'submit',
             showRadar = false,
+            dataTestId,
             ...rest
```

Comment:
I think this should forward `data-testid` to `buttonProps` automatically without needing to pass it in as a separate camelCase prop.